modules/userId/eids.js

```diff
@@ -214,3 +214,24 @@ export function createEidsArray(bidRequestUserId) {
   }
   return eids;
 }
+
+/**
+ * @param {SubmoduleContainer[]} submodules
+ */
+export function buildEidPermissions(submodules) {
+  let eidPermissions = [];
+  submodules.filter(i => utils.isPlainObject(i.idObj) && Object.keys(i.idObj).length && i.config)
+    .forEach(i => {
+      Object.keys(i.idObj).forEach(key => {
+        if (utils.deepAccess(USER_IDS_CONFIG, key + '.source')) {
+          eidPermissions.push(
+            {
+              source: USER_IDS_CONFIG[key].source,
+              bidders: (i.config.bidders && Array.isArray(i.config.bidders) ? i.config.bidders : ['*'])
```

Comment:
since this list of bidders is being sent to SSPs and likely will land in OpenRTB bid requests, should the name `bidders` in the eids object be changed to `pbjs_bidders` since it won't really apply to the openrtb world?

also, rather than `['*']` can we just omit `bidders` (or `pbjs_bidders`) altogether if none were set? This will avoid sending a field that isn't necessary

Reply:
@jdwieland8282

Reply:
@smenzer changing bidders to pbjs_bidders seems fine to me. And yeah I agree omitting `pbjs_bidders` if none are set makes sense. Thanks.